benchmark/url/whatwg-url-properties.js

```diff
@@ -0,0 +1,91 @@
+'use strict';
+
+var common = require('../common.js');
+var URL = require('url').URL;
```

Comment:
These can be `const`. The main reason for keeping `var` in other benchmarks is for compatibility with older versions of Node.js, but since the WHATWG stuff wasn't introduced until v7, it's perfectly fine to use the newer conventions here.